source/solvers/vof.cc

```diff
@@ -455,6 +457,35 @@ template <int dim>
 void
 VolumeOfFluid<dim>::modify_solution()
 {
+  // Peeling/wetting
```

Comment:
the same here. I think a parameter in the parameter handler and an if is required to enable the calculations.